### ISSUE TITLE
Possible to add ability to use KMS over AES?

### ISSUE THREAD

**Issue by pbsladek** [cloudposse/terraform-aws-tfstate-backend#29]
Would like to be able to encrypt via KMS. Wondering if it is possible to add this ability.

https://www.terraform.io/docs/backends/types/s3.html#kms_key_id

**Comment by nitrocode** [comment]
This can be done by overwriting the var `terraform_backend_config_template_file` with your own template file that includes the `kms_key_id`.

```hcl
terraform {
  required_version = ">= ${terraform_version}"

  backend "s3" {
    region         = "${region}"
    bucket         = "${bucket}"
    key            = "${terraform_state_file}"
    dynamodb_table = "${dynamodb_table}"
    profile        = "${profile}"
    role_arn       = "${role_arn}"
    encrypt        = "${encrypt}"
    kms_key_id     = "arn:aws:..."
  }
}
```

**Comment by Gowiem** [comment]
Fixed by #178
